<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.kroy.tests.EnemyTests" tests="7" skipped="0" failures="1" errors="0" timestamp="2020-02-16T19:32:30" hostname="LAPTOP-NPDR84MS" time="0.719">
  <properties/>
  <testcase name="testImproveFortress" classname="com.kroy.tests.EnemyTests" time="0.111"/>
  <testcase name="testCreatePatrol" classname="com.kroy.tests.EnemyTests" time="0.082"/>
  <testcase name="testCalculateTargets" classname="com.kroy.tests.EnemyTests" time="0.098">
    <failure message="java.lang.AssertionError" type="java.lang.AssertionError">java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at com.kroy.tests.EnemyTests.testCalculateTargets(EnemyTests.java:69)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at com.kroy.tests.GdxTestRunner.render(GdxTestRunner.java:59)
	at com.badlogic.gdx.backends.headless.HeadlessApplication.mainLoop(HeadlessApplication.java:129)
	at com.badlogic.gdx.backends.headless.HeadlessApplication$1.run(HeadlessApplication.java:96)
</failure>
  </testcase>
  <testcase name="testDistributesPatrols" classname="com.kroy.tests.EnemyTests" time="0.089"/>
  <testcase name="testCreateFortress" classname="com.kroy.tests.EnemyTests" time="0.083"/>
  <testcase name="testDecideTarget" classname="com.kroy.tests.EnemyTests" time="0.077"/>
  <testcase name="testConstructor" classname="com.kroy.tests.EnemyTests" time="0.081"/>
  <system-out><![CDATA[{}
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
